src/opentimelineio/deserialization.cpp

```diff
@@ -592,7 +592,12 @@ bool deserialize_json_from_string(std::string const& input, any* destination, Er
 }
 
 bool deserialize_json_from_file(std::string const& file_name, any* destination, ErrorStatus* error_status) {
-    FILE* fp = fopen(file_name.c_str(), "r");
+    FILE* fp = nullptr;
+#if defined(_WIN32)
+    fopen_s(&fp, file_name.c_str(), "r");
```

Comment:
From my reading the MSDN docs say that fp will not be changed in the event of an error, but I agree, it would be more clear to test for an error and then set fp to nullptr just in case.